atlas/envs/required_packages.yaml

```diff
@@ -5,7 +5,7 @@ channels:
     - r
 dependencies:
     - python=3.5
-    - bbmap=37.17
+    - bbmap=37.75
```

Comment:
This change is welcomed, but we need to update `bbduk2.sh` command to `bbduk.sh` and verify it works. It appears as though all of the options were added into `bbduk.sh`.

Reply:
According to the [guide](https://jgi.doe.gov/data-and-tools/bbtools/bb-tools-user-guide/bbduk-guide/) bbduk2 can do everything in one go, where we would need several rules with bbduk. what do you think of adding the [bbduk2.sh](https://github.com/BioInfoTools/BBMap/blob/master/sh/bbduk2.sh) script to our script folder. this would also solve the problem.

Reply:
I'm familiar with what it does which is why it's used. The script `bbduk2.sh` is simply a command formatter to build:

```
local CMD="java -Djava.library.path=$NATIVELIBDIR $EA $z $z2 -cp $CP jgi.BBDuk2 $@"
```

Including it without the java class will not work.